Diffing qt_help_bg.ts
Diffing qt_help_el.ts
Diffing qt_help_es.ts
Diffing qt_help_fi.ts
Diffing qt_help_nb.ts
Diffing qt_help_nl.ts
Diffing qt_help_pt.ts
Diffing qt_help_ro.ts
Diffing qt_help_sv.ts
Diffing qt_help_th.ts
Diffing qt_help_tr.ts
qtbase_untranslated.ts qtbase_bg.ts differ: char 22685, line 695
Diffing qtbase_bg.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_bg.ts	2017-02-02 12:37:22.458988925 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Добре</translation>
     </message>
 </context>
 <context>
@@ -903,7 +903,7 @@
     <message>
         <location line="+1"/>
         <source>&amp;OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Добре</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Добре</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Добре</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Отказ</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Отказ</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_da.ts differ: char 139158, line 4106
Diffing qtbase_da.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_da.ts	2017-02-02 12:37:22.502989143 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuller</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuller</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_el.ts differ: char 22685, line 695
Diffing qtbase_el.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_el.ts	2017-02-02 12:37:22.558989421 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ΟΚ</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ΟΚ</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ΟΚ</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ΑΚΥΡΟ</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ΑΚΥΡΟ</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_es.ts differ: char 139158, line 4106
Diffing qtbase_es.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_es.ts	2017-02-02 12:37:22.598989619 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Cancelar</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_fi.ts differ: char 71, line 3
Diffing qtbase_fi.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_fi.ts	2017-02-02 12:37:22.602989639 -0800
     <message>
-        <location line="+2"/>
         <source>&amp;Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Peruuta</translation>
     </message>
     <message>
-        <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Peruuta</translation>
     </message>
qtbase_untranslated.ts qtbase_fr.ts differ: char 139158, line 4106
Diffing qtbase_fr.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_fr.ts	2017-02-02 12:37:22.622989738 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuler</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuler</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_hu.ts differ: char 139158, line 4106
Diffing qtbase_hu.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_hu.ts	2017-02-02 12:37:22.654989897 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Mégse</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Mégse</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_ko.ts differ: char 22685, line 695
Diffing qtbase_ko.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_ko.ts	2017-02-02 12:37:22.726990254 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>확인</translation>
     </message>
 </context>
 <context>
@@ -2677,7 +2677,7 @@
     <message>
         <location line="+286"/>
         <location line="+921"/>
-        <source>OK</source>
+        <translation>확인</translation>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_nb.ts differ: char 139158, line 4106
Diffing qtbase_nb.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_nb.ts	2017-02-02 12:37:22.750990373 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Avbryt</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Avbryt</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_nl.ts differ: char 22685, line 695
Diffing qtbase_nl.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_nl.ts	2017-02-02 12:37:22.778990512 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Oké</translation>
     </message>
 </context>
 <context>
@@ -903,7 +903,7 @@
     <message>
         <location line="+1"/>
         <source>&amp;OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Oké</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Oké</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Oké</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuleren</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annuleren</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_pl.ts differ: char 139158, line 4106
Diffing qtbase_pl.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_pl.ts	2017-02-02 12:37:22.806990650 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Anuluj</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Anuluj</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_pt.ts differ: char 139158, line 4106
Diffing qtbase_pt.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_pt.ts	2017-02-02 12:37:22.834990789 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Cancelar</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_ro.ts differ: char 139158, line 4106
Diffing qtbase_ro.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_ro.ts	2017-02-02 12:37:22.866990948 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Anulează</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Anulează</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_sv.ts differ: char 139158, line 4106
Diffing qtbase_sv.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_sv.ts	2017-02-02 12:37:22.914991186 -0800
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Avbryt</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Avbryt</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_th.ts differ: char 22685, line 695
Diffing qtbase_th.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_th.ts	2017-02-02 12:37:22.946991345 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ตกลง</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ตกลง</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ตกลง</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ยกเลิก</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ยกเลิก</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_tr.ts differ: char 22685, line 695
Diffing qtbase_tr.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_tr.ts	2017-02-02 12:37:22.966991444 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamam</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamam</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamam</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>İptal</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>İptal</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_zh_CN.ts differ: char 22685, line 695
Diffing qtbase_zh_CN.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_zh_CN.ts	2017-02-02 12:37:23.030991761 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>确定</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>确定</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>确定</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>取消</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>取消</translation>
     </message>
 </context>
 <context>
qtbase_untranslated.ts qtbase_zh_TW.ts differ: char 22685, line 695
Diffing qtbase_zh_TW.ts
--- qtbase_untranslated.ts	2017-02-02 13:08:37.768288081 -0800
+++ qtbase_zh_TW.ts	2017-02-02 12:37:23.046991840 -0800
@@ -692,7 +692,7 @@
         <location filename="../../qtbase/src/widgets/dialogs/qmessagebox.cpp" line="+2047"/>
         <location line="+446"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>確定</translation>
     </message>
 </context>
 <context>
@@ -2678,7 +2678,7 @@
         <location line="+286"/>
         <location line="+921"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>確定</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -4043,7 +4043,7 @@
     <message>
         <location filename="../../qtbase/src/gui/kernel/qplatformtheme.cpp" line="+653"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>確定</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4103,7 +4103,7 @@
     <message>
         <location line="+2"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>取消</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4659,7 +4659,7 @@
     <message>
         <location filename="../../qtbase/src/widgets/dialogs/qprogressdialog.cpp" line="+177"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>取消</translation>
     </message>
 </context>
 <context>
